userdocs/src/usage/addons.md

```diff
@@ -74,6 +74,14 @@ You can see what addons are enabled in your cluster by running:
 eksctl get addons --cluster <cluster-name>
 ```
 
+## Setting the addon's version
+
+Setting the version of the addon is optional. If it is not set, `eks` will set the default version for this addon. Note that this might not necessarily be the latest version available. 
```

Comment:
> `eks` will set the default version

eks or eksctl?

Reply:
eks! I'll rephrase this so that it's clearer :)

Reply:
👍  cool, was not sure. have not looked at the addon stuff before :)